tutorials/abap-environment-communication-arrangement/abap-environment-communication-arrangement.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
 title: Maintain a Communication Arrangement for Inbound Communication
-description: Create a communication user and a communication system for an OData service in SAP Cloud Platform ABAP Environment.
+description: Create a communication user and a communication system for an OData service in SAP BTP ABAP Environment.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.